src/ui/mainwindow.cpp

```diff
@@ -0,0 +1,14 @@
+#include "mainwindow.h"
+#include "../ui_mainwindow.h"
+
+MainWindow::MainWindow(QWidget *parent)
+    : QMainWindow(parent)
+    , ui(new Ui::MainWindow)
+{
+    ui->setupUi(this);
```

Comment:
What is setupUI referencing? I don't see it declared in the header or cpp file. 

Also, the naming convention I have been using is pascal case for classes and snake for function/variable/file names. I am not opposed to changing convention, just as long as we remain consistent

Reply:
So it looks like there's a generated header file (In our case here its the ui_mainwindow.h file) when building and setupUi is defined there. I don't think we can change the naming convention for it because its built into Qt